test/system/030-run.bats

```diff
@@ -892,4 +892,14 @@ $IMAGE--c_ok" \
     run_podman container rm -f -t 0 c_ok c_fail_no_rm
 }
 
+@test "podman run --attach stdin prints container ID" {
+    ctr_name="container-$(random_string 5)"
+    run_podman run --name $ctr_name --attach stdin $IMAGE echo hello 2>/dev/null
+    run_output=$output
+    run_podman ps -a --filter name=$ctr_name --format "{{.ID}}"
+    ctr_id=$output
+    is "$run_output" ".*$ctr_id.*" "Did not find container ID in the output"
```

Comment:
We avoid wildcard matches whenever possible: it's too easy to miss warnings or other undesired output. Tight exact matches are best.